### ISSUE TITLE
Summary of issues experienced when deploying to TU BS OCR-D Server

### ISSUE THREAD

**Issue by SvenMarcus** [markusweigelt/kitodo_production_ocrd#6]
- [x] `make build` failed due to a missing slash at the end of `./ocrd/manager/.ssh` on line 28 of Makefile
- [x] ocrd-manager couldn't connect to ocrd-controller due to missing host key of controller
- [x] after adding the host manually, ssh authentication failed from ocrd-manager to controller.
   - can be fixed by providing the `-i ~/.ssh/id_rsa` option for the ssh command in `for_production.sh`
- [x] after running the ocrd workflow once, it fails with `Directory '' already is a workspace` (probably intended behaviour).

**Comment by bertsky** [comment]
@SvenMarcus could you please
- [ ] split up this issue into independent issues
- [ ] provide more details (exactly what you did, commands / logs) to each of the cases, so we can reproduce
- [ ] try to follow the steps detailed in the README

**Comment by SvenMarcus** [comment]
Hm we may have been root instead of ocrd. So the first error is probably a mistake on our end.
As for the-i ~/.ssh/id_rsa issue, there we are pretty sure that we tried to run the workflow by executing the Kitodo script and then we got the authentication error. And only after that we went in to add the ID in order to fix it.

I tried retracing my steps, but after pulling the latest changes I ran into build issues with the Docker services (will create a new issue for this)
